.github/workflows/automationTemplate.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
           
           # run automation again to preserve constants.yaml YamlDotNet indentation
           # since YamlDoNet and yq do not share indentation spacing
-          dotnet run --project build/tools/Automation/Automation.csproj ${RELEASE_DATE} ${{ github.workspace }}
+          dotnet run --project build/tools/Automation/Automation.csproj ${{ inputs.platformName }} ${{ github.workspace }}
```

Comment:
Could you please confirm that this change is on purpose? Could you please share more info about it?

Reply:
Yes, this is correct. It wasn't a big deal since it just helps us correct the spacing issue, as noted in the comment above. The correct parameter should be the platformName not the RELEASE_DATE